edweiss-app/components/memento/CardScreenComponent.tsx

```diff
@@ -0,0 +1,247 @@
+                        icon='close-sharp'
+                    >
+                        Not yet
+                    </FancyButton>
+                </TView>
+                <TView style={styles.lastButtonHalf}>
+                    <FancyButton
+                        backgroundColor='green'
+                        onPress={() => updateLearningStatusCard(deckId, cardIndex, updateCard, "Got it", card)}
+                        style={{ width: '100%', height: '100%' }} // Make button fill its half
+                        icon='checkmark-sharp'
+                    >
+                        Got it!
+                    </FancyButton>
+                </TView>
+            </TView>
+
+        </TView >
+    );
+
+};
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginBottom: '20%'
+    },
+    modalContainer: {
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginBottom: '20%',
+    },
+    modalCard: {
+        width: '70%',
+        height: '75%',
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: 'white',
+        borderRadius: 20,
+        borderColor: 'crust',
+        backfaceVisibility: 'hidden',
+        top: '-2%',
+    },
+    cardContainer: {
+        width: '70%',
+        height: '75%',
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: 'white',
+        borderRadius: 20,
+        borderColor: 'crust',
+        backfaceVisibility: 'hidden',
+        position: 'absolute',
+        top: '10%',
+    },
+    buttonContainer: {
+        position: 'absolute',
+        bottom: '0%', // Place the row near the bottom
+        width: '60%', // Keep the row aligned with the card width
+        flexDirection: 'row', // Create a horizontal row
+        justifyContent: 'space-between',
+        paddingHorizontal: 0,
+        borderWidth: 1, // Add border to mimic "table" look (optional)
+        borderColor: 'transparent', // Border color (optional)
+        borderRadius: 20,
+    },
+    buttonHalf: {
+        flex: 1, // Take up half the width for each button
+        justifyContent: 'center',
+        alignItems: 'center',
+        borderRightWidth: 5, // Add a border between the two buttons (optional)
+        borderColor: 'gray',
+        height: 50, // Set height of the button row
+    },
+    lastButtonHalf: {
+        flex: 1, // Take up half the width for the second button
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: 50,
+    }
+});
+
+export default CardScreenComponent;
+
+function updateLearningStatusCard(deckId: string, cardIndex: number, onPress: (deckId: any, newCard: Memento.Card, cardIndex: number) => void, learning_status: Memento.LearningStatus, card?: Memento.Card) {
+    if (card) {
+        card.learning_status = learning_status;
```

Comment:
This a good approach, I would like to propose another on
`const updatedCard = { ...card, learning_status };`
Again this is a question of taste, do it as you feel.

Reply:
Ok, thank youu!!